### PR TITLE
Updated gitlab-ci file, and removed Jenkinsfile.

### LINKED ISSUE #13: port fluent-bit to gitlab
- [x]  Follow [instructions](https://github.com/samsung-cnct/solas/blob/master/docs/gitlab.md) to create  and edit`.gitlab-ci.yaml` at project root from [template](https://github.com/samsung-cnct/solas-container/blob/master/.gitlab-ci.yml), as well as create gitlab secret variable, and add pipeline status badge to README
- [x] Remove the `Jenkinsfile` in the current repo 
- [ ] When PR has merged, make sure that new image is being created on quay. Afterwards,  please head back to the [umbrella ticket](https://github.com/samsung-cnct/issues/issues/89) and check off the `container-fluent-bit`

### PR DESCRIPTION
**Pull request by yenicapotediaz**
Updated gitlab-ci file, and removed Jenkinsfile.
Fixes #13

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -10,21 +10,22 @@ before_script:
...
   IMAGE_PROD_NAME: $CI_REGISTRY_IMAGE:$CI_COMMIT_REF_NAME
-  registry: quay.io
-  image_name: fluent-bit-container
-  quay_org: samsung_cnct
-  QUAY_ROBOT: SECRET 
-  QUAY_PASSWORD: SECRET
+  REGISTRY: quay.io
+  IMAGE_NAME: fluent-bit-container
+  REGISTRY_USER: your_org # we set this to override to samsung_cnct in gitlab Group Variables
+  ROBOT_ACCOUNT: fluent_bit_container
```
Comment (davidewatson) [inline .gitlab-ci.yml]: According to the solas [GitLab](https://github.com/samsung-cnct/solas/blob/master/docs/gitlab.md#edit-gitlab-ciyml-file) docs this should be `fluent_bit_container_rw`

Comment (guineveresaenger) [reply .gitlab-ci.yml]: The robot name on quay is wrong, then. I suppose that means update the robot/ create a new one?

Comment (yenicapotediaz) [reply .gitlab-ci.yml]: The current robot account there is called fluent_bit_container (you created it). Should I a) rename it to fluent_bit_container_rw or b) create a new robot account?

Comment (yenicapotediaz) [reply .gitlab-ci.yml]: I will proceed with b

**Pull request merged**